### PR TITLE
moves common models to common module for use in reports

### PR DESCRIPTION
**Pull request by esotrope**

### CODE REVIEW

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/jdbc/ScaleScores.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import java.sql.SQLException;
+
+import static org.opentestsystem.rdw.reporting.common.jdbc.QueryUtils.getNullable;
+
+/**
+ * Holds utility methods for mapping SQL query results to {@link ScaleScore}
+ */
+public final class ScaleScores {
+
+    private ScaleScores() {
```
Comment (malaffoon) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/jdbc/ScaleScores.java]: nit (and probably a personal preference, not an absolute guideline): i like "hiding" the private ctor at the bottom of utility classes, just so you don't have to read past it.

File: webapp/src/test/java/org/opentestsystem/rdw/reporting/search/user/DefaultUserContextServiceTest.java [hunk 2]
```diff
@@ -2,13 +2,11 @@
 
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Maps;
```
Comment (malaffoon) [inline webapp/src/test/java/org/opentestsystem/rdw/reporting/search/user/DefaultUserContextServiceTest.java]: i'm assuming this is unrelated cmd-alt-O cleanup?

File: common/src/main/java/org/opentestsystem/rdw/reporting/common/jdbc/ScaleScores.java (hunk 1 above)
Comment (esotrope) [inline common/src/main/java/org/opentestsystem/rdw/reporting/common/jdbc/ScaleScores.java]: Our intellij autoformat settings like it there though : J

File: webapp/src/test/java/org/opentestsystem/rdw/reporting/search/user/DefaultUserContextServiceTest.java (hunk 2 above)
Comment (esotrope) [inline webapp/src/test/java/org/opentestsystem/rdw/reporting/search/user/DefaultUserContextServiceTest.java]: This is interesting. I don't think I made this change but somehow intellij did when I dragged files around.
